website/docs/r/linux_web_app.html.markdown

```diff
@@ -411,6 +411,8 @@ A `site_config` block supports the following:
 
 * `always_on` - (Optional) If this Linux Web App is Always On enabled. Defaults to `true`.
 
+~> **NOTE:** `always_on` cannot be set to `true` when using `Free`, `F1`, `D1` SKU, it has to be turned off before switching to Free or Shared SKU.
```

Comment:
can we not do the SKU update before doing the always on update so the user doesn't have to run terraform multiple times?

Reply:
For clarity, the SKU is the Service Plan SKU, not on the App Service, so it will be updated first if both changes are made in config and the resource dependency is appropriately set (e.g. reference the ID of the service plan in the app), and would be a single `apply`. However, if the Service Plan SKU is updated without the `always_on` property being appropriately set on the App, the API call to set to `Free` or `shared` will fail. There's no way to avoid / guard-rail this for the user unfortunately, being 2 separate (if dependent) resources. 

I think the note should possibly read:
```suggestion
~> **NOTE:** `always_on` must be explicitly set to `false` when using `Free`, `F1`, `D1`, or `Shared` Service Plans.
```